pkg/command/context.go

```diff
@@ -94,21 +100,36 @@ var createCtxCmd = &cobra.Command{
 	Args:  cobra.MaximumNArgs(1),
 	RunE:  createCtx,
 	Example: `
+    # Create a TKG management cluster context using endpoint and type(--type is optional, if not provided CLI would infer the type from the endpoint)
+    tanzu context create mgmt-cluster --endpoint "https://k8s.example.com[:port] --type k8s-cluster-endpoint"
+
     # Create a TKG management cluster context using endpoint
     tanzu context create mgmt-cluster --endpoint "https://k8s.example.com[:port]"
 
     # Create a TKG management cluster context using kubeconfig path and context
     tanzu context create mgmt-cluster --kubeconfig path/to/kubeconfig --kubecontext kubecontext
 
     # Create a TKG management cluster context by using the provided CA Bundle for TLS verification:
-    tanzu context create --endpoint https://k8s.example.com[:port] --endpoint-ca-certificate /path/to/ca/ca-cert
+    tanzu context create mgmt-cluster --endpoint https://k8s.example.com[:port] --endpoint-ca-certificate /path/to/ca/ca-cert
 
     # Create a TKG management cluster context by explicit request to skip TLS verification, which is insecure:
-    tanzu context create --endpoint https://k8s.example.com[:port] --insecure-skip-tls-verify
+    tanzu context create mgmt-cluster --endpoint https://k8s.example.com[:port] --insecure-skip-tls-verify
 
     # Create a TKG management cluster context using default kubeconfig path and a kubeconfig context
     tanzu context create mgmt-cluster --kubecontext kubecontext
 
+   # Create a Application Engine(UCP) context with default endpoint (--type option is not necessary for default endpoint)
+    tanzu context create myucp -endpoint https://api.tanzu.cloud.vmware.com 
+
+    # Create a Application Engine(UCP) context (--type option is needed for non-default endpoint)
+    tanzu context create myucp -endpoint https://non-default.ucp.endpoint.com --type application-engine
+
+    # Create a Application Engine(UCP) context by using the provided CA Bundle for TLS verification:
+    tanzu context create myucp --endpoint https://api.tanzu.cloud.vmware.com  --endpoint-ca-certificate /path/to/ca/ca-cert
+
+    # Create a Application Engine(UCP) context by explicit request to skip TLS verification, which is insecure:
+    tanzu context create myucp --endpoint https://api.tanzu.cloud.vmware.com --insecure-skip-tls-verify
+
```

Comment:
preexisting: we should add example for mission-control as well, but I expect we will have to do another pass on the examples, we can do this in the followup.